src/app/components/Persist/LoginModalLayout.tsx

```diff
@@ -3,8 +3,8 @@ import { ResponsiveLayer } from '../ResponsiveLayer'
 import React, { useContext } from 'react'
 import { Header } from 'app/components/Header'
 import { Navigation } from '../Sidebar'
-import walletBlueLogotype from '../../../../public/Rose Wallet Blue.svg'
-import walletWhiteLogotype from '../../../../public/Rose Wallet White.svg'
+import walletBlueLogotype from 'url:../../../../public/Rose Wallet Blue.svg'
+import walletWhiteLogotype from 'url:../../../../public/Rose Wallet White.svg'
```

Comment:
+1, looks like `"*.svg": ["...", "@parcel/transformer-raw"],` will work the same